SPECS/libksba/libksba.spec

```diff
@@ -1,51 +1,67 @@
-Summary:	Library for accessing X.509 and CMS data structure.
-Name:		libksba
-Version:	1.3.5
-Release:        4%{?dist}
-License:	GPLv3+
-URL:		https://www.gnupg.org/(fr)/download/index.html#libksba
-Group:		Security/Libraries.
-Source0:        https://www.gnupg.org/ftp/gcrypt/%{name}/%{name}-%{version}.tar.bz2
+Summary:        Library for accessing X.509 and CMS data structure.
+Name:           libksba
+Version:        1.6.0
+Release:        1%{?dist}
+# See AUTHORS file for licensing details
+License:	    (LGPLv3+ or GPLv2+) and GPLv3+ 
```

Comment:
Nit: let's lint this file as well and remove tabs.

Reply:
Thanks for catching those- fixed!